etd_app/forms.py

```diff
@@ -25,7 +25,9 @@ class Meta:
             }
         help_texts = {
                 'first_name': 'Must match name on thesis or dissertation',
-                'last_name': 'Must match name on thesis or dissertation'
+                'last_name': 'Must match name on thesis or dissertation',
+                'orcid': 'Won\'t be published in BDR',
```

Comment:
Is the BDR mentioned anywhere else in the application?
Can we assume they know what the BDR is?
Is it enough to say that these fields won't be published?